app/src/main/java/com/github/fribourgsdp/radio/Genre.kt

```diff
@@ -23,7 +23,7 @@ enum class Genre (private val genreName: String){
     MOVIE("Movie Music"),
     COUNTRY("Country");
 
-    override fun toString(): String {
+    /*override fun toString(): String {
```

Comment:
I understand this is necessary the cloud, but could you then rename this function here and wherever it is called ? so we do not overwrite tostring but we do not remove a functionnality

Reply:
Yes of course